io.dbeaver.DBeaverCommunity.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.dbeaver.DBeaverCommunity</id>
+    <launchable type="desktop-id">io.dbeaver.DBeaverCommunity.desktop</launchable>
+    <metadata_license>ALS</metadata_license>
+    <project_license>LicenseRef=https://dbeaver.io/files/dbeaver_license.txt</project_license>
+    <name>DBeaver Community</name>
+    <url type="homepage">https://dbeaver.io</url>
+    <summary>DBeaver Community. Universal Database Tool.</summary>
+    <description>
+        <p>Free multi-platform database tool for developers, SQL programmers, database administrators and analysts. Supports all popular databases: MySQL, PostgreSQL, MariaDB, SQLite, Oracle, DB2, SQL Server, Sybase, MS Access, Teradata, Firebird, Derby, etc.</p>
+        <p>This is the official build of DBeaver, packaged into a Flatpak. This repackaging is not supported by DBeaver Community Edition.</p>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://dbeaver.io/wp-content/uploads/2018/03/data_edit-450x220.png</image>
+            <caption>Data Edit</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://dbeaver.io/wp-content/uploads/2018/03/sql_script-450x220.png</image>
+            <caption>SQL Script Editor</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://dbeaver.io/wp-content/uploads/2018/03/erd-450x220.png</image>
+            <caption>ERD</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://dbeaver.io/wp-content/uploads/2018/03/mock_data-450x220.png</image>
+            <caption>Mock Data Generator</caption>
+        </screenshot>
+    </screenshots>
+    <releases>
+        <release version="5.2.3" date="2018-10-21"/>
```

Comment:
Since you are actually using release `5.2.4` it would be nice to change it here also :innocent: